wrangler-core/src/main/java/co/cask/wrangler/steps/RecordRegexFilter.java

```diff
@@ -28,6 +29,7 @@
 /**
  * A Wrangle step for filtering rows that match the pattern specified on the column.
  */
+@Usage(directive = "filter-row-if-matched", usage = "filter-row-if-matched <column> <regex>")
```

Comment:
is there a filter-column-if-matched? if not, can these just be filter-if-matched?